.github/workflows/semantic.yml

```diff
@@ -2,7 +2,7 @@ name: semantic-release
 
 on:
   push:
-    branches: []
+    branches: [master]
```

Comment:
@gustavobeavis I changed this one because the semantic release must run only when changes are merged to master